lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
close but you have to use `.call` instead of `.bind`. bind returns a bound function, but doesn't call it. call binds and calls the function